R/TuneWrapper.R

```diff
@@ -75,7 +75,10 @@ trainLearner.TuneWrapper = function(.learner, .task, .subset = NULL,  ...) {
 #' @export
 predictLearner.TuneWrapper = function(.learner, .model, .newdata, ...) {
   lrn = setHyperPars(.learner$next.learner, par.vals = .model$learner.model$opt.result$x)
-  predictLearner(lrn, .model$learner.model$next.model, .newdata, ...)
+  arglist = list(.learner = lrn, .model = .model$learner.model$next.model, .newdata = .newdata)
+  arglist = insert(arglist, list(...))
+  arglist = insert(arglist, .model$learner.model$opt.result$x)
```

Comment:
Should probably only insert the parameters that have `when %in% c("predict", "both")`.

Maybe one could just copy [the lines from predict.R](https://github.com/mlr-org/mlr/blob/e547bae5e776c9378a764e33962f82cb44f95ecd/R/predict.R#L101-L106), if we are sure there are no other wrappers that introduce a discrepancy between the `par.vals` of a learner and the `...` parameters on purpose.